rune/utils_linux.go

```diff
@@ -305,6 +305,12 @@ func (r *runner) run(config *specs.Process) (int, error) {
 	var (
 		detach = r.detach || (r.action == CT_ACT_CREATE)
 	)
+
+	detachFlag := int(0)
+	if detach {
+		detachFlag = 1
+	}
+	process.Env = append(process.Env, fmt.Sprintf("_LIBENCLAVE_DETACH=%d", detachFlag))
```

Comment:
The variable inserted here will be persistent. Actually we only need a temporary setting passed to init-runelet. Follow the way of what commandTemplate() does. And the variable should be named as _LIBCONTAINER_DETACHED and than staged to _LIBENCLAVE_DETACHED and finally seen by init-runelet in StartInitialization().